### PR TITLE
Feature combine radar apps

### PR DESCRIPTION
**Pull request by mollybsmith-noaa**
My branch with the combined radar app. I'd like to get it on to development to see if it builds and passes automated tests. Manual testing went perfectly.

### CODE REVIEW

**Comment by ian-noaa** [comment]
I'll take a deeper look at this tomorrow since there are a lot of changes. My first two thoughts are that we should fix CI (I'm happy to help with that! I suspect the app list in `.github/workflows/build.yml` and in `deploy.yml` need to be updated to match the new app names) and that we may need to update the test suite.

**Comment by mollybsmith-noaa** [comment]
Ah, okay. I updated the app list in build.sh but missed those two files.
Sorry!

**Review by randytpierce — APPROVED** [review]
This all looks straight forward, but did it affect the webdriver tests? Are those updated to reflect these changes?

**Comment by ian-noaa** [comment]
It looks like the webdriver tests were updated so thanks for doing that Molly! Would it be a good idea to add tests to check the other database parameters? I'm not sure if there are other code paths that using different databases would exercise. 

Regarding CI - you'll need to update the `matrix:app` list in `.github/workflows/build.yml` and the `matrix:deploy` list in `.github/workflows/deploy.yml` with the new app names.

Otherwise, I'm assuming the rest of the changes are good.

**Comment by mollybsmith-noaa** [comment]
The webdriver tests are indeed included in this update.

**Comment by mollybsmith-noaa** [comment]
Also Ian-- thanks for pointing me towards the build.yml and deploy.yml, I'll get those updated!

**Comment by mollybsmith-noaa** [comment]
@ian-noaa It tried to build the correct apps this time but doesn't seem to have succeeded. Did I miss a necessary step for this?

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ FROM geoffreybooth/meteor-base:2.5 AS meteor-builder
 
 ARG APPNAME
 
+RUN apt update && apt install -y ssh-client
+
```
Comment (ian-noaa) [inline Dockerfile:7]: Lets remove these two lines - they're no longer needed now that we're using jspdf 2.5.

Comment (mollybsmith-noaa) [reply Dockerfile]: Done!

**Pull request merged**